src/NuGet.Clients/PackageManagement.UI/Resources.resx

```diff
@@ -511,4 +511,7 @@ Packages installed into Website projects will not be restored during build. Cons
     <value>not installed</value>
     <comment>In the solution package manager, if the select package is not installed, then this message is shown.</comment>
   </data>
+  <data name="Label_Consolidate" xml:space="preserve">
+    <value>Consolidate</value>
```

Comment:
This also needs localization, I believe we already have this string elsewhere today, can we reuse?

CC @RanjiniM